hazelcast/src/main/java/com/hazelcast/sql/impl/row/ListRowBatch.java

```diff
@@ -42,8 +42,6 @@ public ListRowBatch(List<Row> rows) {
 
     @Override
     public Row getRow(int index) {
-        assert index >= 0 && index < rows.size() : index;
```

Comment:
What was the reason to remove this assertion?

Reply:
It's superfluous, the next line does the same check anyway.